### DISCUSSION TITLE
post/must_reads/

### DISCUSSION THREAD

**Discussion by swilcox** [swilcox/swilcox.github.io#6] [General]
https://swilcox.github.io/post/must_reads/ 

 There is a blog (not mine) that you need to be reading!

**Comment by bcorfman** [comment]
I spent a little time reading his AI content. How did I not know about Claude's Artifacts feature?? What the heck. I feel rather out of the loop right now. I even had to spend a little time asking Claude "What am I supposed to use this for?" because I was scratching my head at first.  :joy:
